src/Encrypt/Openssl.php

```diff
@@ -364,11 +364,17 @@ public function encrypt($value)
             $value    = $compress($value);
         }
 
-        $crypt  = openssl_seal($value, $encrypted, $encryptedkeys, $keys);
-        foreach ($keys as $key) {
-            openssl_free_key($key);
```

Comment:
Please just change this function call to just target the new created method `Openssl:: freeKeyResource` to keep maintainability easy.